API.md

```diff
@@ -0,0 +1,795 @@
+# PDFTron Flutter API
+
+- [Utility Function APIs](#Utility-Function-APIs)
+- [Viewer Function APIs](#Viewer-Function-APIs)
+- [Event APIs](#Event-APIs)
+- [Viewer Configurations](#Viewer-Configurations)
+
+## Utility Function APIs
```

Comment:
i don't think it was called like this for React Native? maybe just "Methods"

Reply:
For React Native, there is only RNPdftron and the viewer component. But here, I also wish to separate the static utility functions and the other viewer-related functions,